draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -726,40 +768,48 @@ It is up to the Holder how to maintain the mapping between the Disclosures and t
 
 ## Processing by the Holder  {#holder_verification}
 
-If a Key Binding JWT is received by a Holder, the SD-JWT SHOULD be rejected.
+The Issuer MUST provide the Holder an SD-JWT, not an SD-JWT-KB.  If the Holder
+receives an SD-JWT-KB from the Issuer, it SHOULD be rejected.
 
 For presentation to a Verifier, the Holder MUST perform the following (or equivalent) steps:
 
  1. Decide which Disclosures to release to the Verifier, obtaining proper End-User consent if necessary.
- 2. Assemble the SD-JWT for presentation, including the Issuer-signed JWT and the selected Disclosures (see (#data_formats) for the format).
- 3. If Key Binding is required, create a Key Binding JWT ((#kb-jwt)) and add it to the SD-JWT for presentation.
- 4. Send the SD-JWT for presentation to the Verifier.
+ 2. Assemble the SD-JWT, including the Issuer-signed JWT and the selected Disclosures (see (#data_formats) for the format).
+ 3. If Key Binding is not required:
+    1. Send the SD-JWT to the Verifier.
+ 4. If Key Binding is required:
+    1. Create a Key Binding JWT covering the SD-JWT.
+    2. Assemble the SD-JWT-KB, including the SD-JWT and the Key Binding JWT.
+    3. Send the SD-JWT-KB to the Verifier.
 
 ## Verification by the Verifier  {#verifier_verification}
 
-Upon receiving a Presentation, in addition to the checks outlined in (#sd_jwt_verification), Verifiers need to ensure that
+Upon receiving a presentation from a Holder, in the form of either an SD-JWT or
+an SD-JWT-KB, in addition to the checks outlined in (#sd_jwt_verification), Verifiers MUST ensure that
```

Comment:
We avoided normative language here in this "summary" in order to not duplicate the normative requirement defined below.
```suggestion
an SD-JWT-KB, in addition to the checks outlined in (#sd_jwt_verification), Verifiers need to ensure that
```